templates/dist-datastore.cfg.xml.erb

```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <!-- http://wiki.openrepose.org/display/REPOSE/Distributed+Datastore -->
-<distributed-datastore xmlns="http://<%= scope.lookupvar('::repose::cfg_namespace_host_dist') %>/repose/distributed-datastore/v1.0">
+<distributed-datastore xmlns="http://<%= scope.lookupvar('::repose::cfg_namespace_host_dist') %>/repose/distributed-datastore/v1.0"
+<% if @connection_pool_id -%>
+                       connection-pool-id="<%= @connection_pool_id %>"
+<% end -%>>
```

Comment:
I would perhaps put the close container on the next line to not confuse it with the non-printing erb templating.